src/features/views/components/ViewDataTable/columnTypes/SurveyResponseColumnType.tsx

```diff
@@ -38,7 +38,7 @@ export default class SurveyResponseColumnType
       },
       valueGetter: (params: GridValueGetterParams) => {
         const cell: SurveyResponseViewCell = params.row[params.field];
```

Comment:
I guess we are sort of lying here, saying that `cell` will always be a `SurveyResponseViewCell` when actually it could also be `undefined` , as we see on the next row with the `?`. But I'm approving just because we need the fix more than we need the ultra perfect typescript ;)